src/plugins/navigation_embeddable/public/components/editor/navigation_embeddable_panel_editor.tsx

```diff
@@ -215,13 +201,29 @@ const NavigationEmbeddablePanelEditor = ({
                       ))}
                     </EuiDroppable>
                   </EuiDragDropContext>
-                  <EuiButtonEmpty size="s" iconType="plusInCircle" onClick={() => addOrEditLink()}>
+                  <EuiButtonEmpty
+                    flush="left"
+                    size="s"
+                    iconType="plusInCircle"
+                    onClick={() => addOrEditLink()}
+                  >
                     {NavEmbeddableStrings.editor.getAddButtonLabel()}
                   </EuiButtonEmpty>
-                </>
-              )}
+                </div>
+              </EuiFormRow>
+              <EuiFormRow label={NavEmbeddableStrings.editor.panelEditor.getLayoutSettingsTitle()}>
```

Comment:
I wonder if we should move the layout options above the links? If some intrepid person adds 50 links to a panel then they have to scroll all the way down to change the layout. Maybe that won't ever happen, but I feel like putting a known number of form controls (`1`) on top of an unknown number of controls (`1 -->  ∞`) is better UI. 

We could also paginate long lists of links, but that seems unnecessary.

Reply:
That's a really good point - here's what that would look like:

![image](https://github.com/elastic/kibana/assets/8698078/31e1ec43-bb01-4818-a32a-17733ed2411e)

I don't mind it 🤔 What do you think, @andreadelrio?

Reply:
I think this makes sense 👍

Reply:
Done in db2439f4aa26d95bcdce21f4c8d0008ffbf2b0cb 🎉